src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -89,7 +89,7 @@
                     Visibility="{Binding CheckBoxesEnabled, RelativeSource={RelativeSource AncestorType={x:Type nuget:InfiniteScrollList}}, Converter={StaticResource NonVisualStateHiddenBooleanToVisibilityConverter}}"
                     Focusable="False"
                     AutomationProperties.Name="{x:Static nuget:Resources.CheckBox_Selected}"
-                    IsChecked="{Binding Selected}"
+                    IsChecked="{Binding IsSelected}"
```

Comment:
File can be reverted if rename is undone

Reply:
This change is necessary for the reusability of the automation peer. Reverting the name change would make the automation peer not reusable. Please see my comment above about the rename in PackageItemListViewModel.cs.